src/mon/CreatingPGs.h

```diff
@@ -0,0 +1,26 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <map>
+#include <unordered_set>
+#include "include/encoding.h"
+
+struct creating_pgs_t {
+  std::map<pg_t, std::pair<epoch_t, utime_t> > pgs;
+  std::unordered_set<int64_t> created_pools;
```

Comment:
@liewegas unordered_set is more spatial efficient in this case, as pool id are sequential numbers which are a perfect fit for the hash.